### ISSUE TITLE
 Increased serum thymidine - Increased serum deoxyuridine 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7538]
**Preferred term label:**
 Increased serum thymidine
Increased serum deoxyuridine 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
603041 
PMID: 14757860  1/1 for both 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
